package.json

```diff
@@ -10,31 +10,37 @@
   "jsnext:main": "dist/es/index.js",
   "license": "MIT",
   "scripts": {
-    "build": "yarn run build:commonjs && yarn run build:css && yarn run build:es && yarn run build:demo && yarn run build:umd",
-    "build:commonjs": "yarn run clean:commonjs && cross-env NODE_ENV=production cross-env BABEL_ENV=commonjs babel source --out-dir dist/commonjs --ignore *.example.js,*.test.js,source/demo/,source/tests.js",
+    "build": "npm run build:commonjs && npm run build:css && npm run build:es && npm run build:demo && npm run build:umd",
```

Comment:
It should, but I'm trying to debug [this](https://circleci.com/gh/bvaughn/react-virtualized/1514) failure:
```
$yarn
bash: line 1: yarn: command not found

yarn returned exit code 127

Action failed: yarn
```

Just trying different things out. Not sure why it's failing. ☹️ Tried upgrading Circle from Ubuntu 12 to 14 and that didn't help. Tried modifying my circle.yml as shown in the examples. Ran out of ideas so I tried this.